periodic/periodic.go

```diff
@@ -406,7 +408,8 @@ func (r *periodicRunner) Run() RunnerResults {
 	}
 	start := time.Now()
 	// Histogram  and stats for Function duration - millisecond precision
-	functionDuration := stats.NewHistogram(0, r.Resolution)
+	offsetSec := float64(r.Offset) / float64(time.Second)
```

Comment:
https://pkg.go.dev/time#Duration.Seconds

Reply:
thanks. It is much easier to get the seconds.  Code updated.